README.md

```diff
@@ -1,4 +1,152 @@
+The `React` frontend is responsible for the interfaces with which users interact in the LTI Iframes. It gets an authenticated token with permissions
+from the view and interacts with the `Django` backend to manage objects and directly with `AWS s3` to upload files.
+
+It also powers the same view when loaded by a learner to display a video player (thanks to [Plyr](https://plyr.io)).
+
+⚠️ **Iframe management**
+
+To have the best possible user experience for instructors, we need to be able to change the size of the `<iframe>` depending on its contents. This can be done through the [iframe-resizer](https://github.com/davidjbradshaw/iframe-resizer) library.
+
+`iframe-resizer` requires to run some JS inside the `<iframe>` (which we include with our `React` frontend bundle) and some JS inside the host page. It then communicates through message-passing to adjust the size of the `<iframe>`.
+
+This means that to have the best interfaces for instructors, you need to include the host-side `iframe-resizer` JS in your LMS pages. This is already done in our [custom LTI consumer Xblock for Open edX](https://github.com/openfun/xblock-configurable-lti-consumer).
+
+If you cannot or do not want to include this host-side JS, you can still run `Marsha`. It will work exactly the same for learners (provided you adjust the size of the LTI `<iframe>` for video), and instructors will only have to scroll inside the `<iframe>` on some occasions.
+
+## Getting started
+
+Make sure you have a recent version of Docker and
+[Docker Compose](https://docs.docker.com/compose/install) installed on your laptop:
+
+```bash
+$ docker -v
+  Docker version 18.09.0, build 4d60db4
+
+$ docker-compose --version
+  docker-compose version 1.23.2, build 1110ad01
+```
+
+⚠️ You may need to run the following commands with `sudo` but this can be avoided by assigning your user to the `docker` group.
+
+### The `Django` Backend
+
+All tasks related to the `Django` backend are run from the project root (where this `README.md` is located).
+
+The easiest way to start working on the project is to use our `Makefile`:
+
+    $ make bootstrap
+
+This command builds the `app` container, installs back-end dependencies and performs database migrations. It's a good idea to use this command each time you are pulling code from the project repository to avoid dependency-related or migration-related issues.
+
+🔧 **Before you go further**, you should take a look at the newly created `./env.d/development` file and replace the relevant values with your own. You can take a look at the [environment documentation](https://github.com/openfun/marsha/blob/write-the-readme/docs/env.md#1-django-backend-environment) for more details on this topic.
+
+Now that your `Docker` services are ready to be used, start the application by running:
+
+    $ make run
+
+You should be able to view the development view at [localhost:8060/development/](http://localhost:8060/development/).
+
+### The storage & transcoding environment
```

Comment:
I think AWS needs to come first because you use the output from Terraform to configure your backend.